### PR TITLE
Simplify inventory REST tutorial

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

#7545 updated the Inventory REST tutorial to be compatible with the new in-store pickup feature, but it was causing some problems, and I couldn't complete the GraphQL ISP tutorial. This PR simplifies the REST tutorial by performing all steps on the `default` store view. As a side effect of this, I replaced all references to German warehouses/stores with entities in California 

I added a substep to explain where to get order item IDs. I also deleted a substep that featured the `/V1/shipment` endpoint, which is obsolete.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/rest/tutorials/inventory/index.html

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.4/rest/tutorials/inventory/create-customer.md [hunk 1]
```diff
@@ -13,82 +13,19 @@ functional_areas:
...
-        "is_active": 1
-    }
-]
-```
+This step creates a customer that lives close to several stores that serve as in-store pickup locations. Although this tutorial ships the order using a traditional shipping method, the GraphQL tutorial shows how in-store pickup can be implemented.
 
 ## Create a customer
 
-We can use the same customer definition presented in [Step 3. Create a customer]({{ page.baseurl }}/rest/tutorials/orders/order-create-customer.html), with the only differences being the scope of the call (`us` instead of `default`) and inserting the `store_id` and `website_id` parameters into the payload.
+If you prefer to use a customer is already defined on your installation, scroll down to [Generate the customer's access token](#get-token)
```
Comment (dobooth) [inline src/guides/v2.4/rest/tutorials/inventory/create-customer.md:20]: "...customer that is..."

File: src/guides/v2.4/rest/tutorials/inventory/create-shipment.md [hunk 2]
```diff
@@ -13,31 +13,30 @@ functional_areas:
   - Integration
 ---
 
-In the previous step, the SSA recommended shipping 35 `24-WB01` items and 20 `24-WB03` items from the Northeast warehouse. It also recommended shipping 5 `24-WB01` items from the Brooklyn store.
+In the previous step, the SSA recommended shipping 35 `24-WB01` items and 20 `24-WB03` items from the Northeast warehouse. It also recommended shipping 5 `24-WB01` items from the West warehouse.
 
 You can always override the SSA recommendations. It would be valid, for example, to ship 25 `24-WB01` items and 5 `24-WB03` items each from the Brooklyn, Manhattan, and Huntington stores.
 
-Fulfilling the order requires two partial shipments (unless you want to create a scenario that also involves more than one store). In this example, we'll follow the SSA recommendations.
+Fulfilling the order requires a minimum of two partial shipments. In this example, we'll follow the SSA recommendations.
```
Comment (dobooth) [inline src/guides/v2.4/rest/tutorials/inventory/create-shipment.md:20]: we'll -> we will

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**